lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -17,11 +17,21 @@ def self.validate_ios_app(file_path)
             size = conf.size
             if conf['__xctestrun_metadata__']
               size -= 1
-            end  
+            end
             unless size == 1
-              UI.user_error!("The app bundle may contain only one scheme, #{size} found")
+              UI.user_error!("The app bundle may contain only one scheme, #{size} found.")
+            end
+
+            scheme_conf = nil
+            conf.each do |key, value|
+              if scheme_conf.nil? && key != '__xctestrun_metadata__'
+                scheme_conf = value
+              end
+            end
+
+            unless !scheme_conf.nil?
```

Comment:
Almost there! Last comment here: let's use if versus "unless !".

Reply:
Done.

Out of curiosity though, what's the difference?